### PR TITLE
Meta ids

### PR DESCRIPTION
**Pull request by micbar**
# Description

Add IDs to `dav/meta` response.

### CODE REVIEW

File: internal/http/services/owncloud/ocdav/meta.go [hunk 1]
```diff
@@ -142,6 +142,9 @@ func (h *MetaHandler) handlePathForUser(w http.ResponseWriter, r *http.Request,
 		Status: "HTTP/1.1 200 OK",
 		Prop: []prop.PropertyXML{
 			prop.Escaped("oc:meta-path-for-user", pathRes.Path),
+			prop.Escaped("oc:id", id),
+			prop.Escaped("oc:fileid", id),
+			prop.Escaped("oc:spaceid", rid.StorageId),
```
Comment (kobergj) [inline internal/http/services/owncloud/ocdav/meta.go:147]: Could possibly run into panics if `rid` is `nil`. Use `rid.GetStorageId()` to avoid that

**Review by kobergj — CHANGES_REQUESTED** [review]

**Review by kobergj — APPROVED** [review]

File: internal/http/services/owncloud/ocdav/meta.go [hunk 2]
```diff
@@ -142,6 +142,9 @@ func (h *MetaHandler) handlePathForUser(w http.ResponseWriter, r *http.Request,
 		Status: "HTTP/1.1 200 OK",
 		Prop: []prop.PropertyXML{
 			prop.Escaped("oc:meta-path-for-user", pathRes.Path),
+			prop.Escaped("oc:id", id),
+			prop.Escaped("oc:fileid", id),
+			prop.Escaped("oc:spaceid", rid.GetStorageId()),
```
[the remark this answers is not in the document]
Comment (micbar) [reply internal/http/services/owncloud/ocdav/meta.go:147]: naming 😄

[the remark this answers is not in the document]
Comment (dragotin) [reply internal/http/services/owncloud/ocdav/meta.go:147]: ...can and should be changed to make the world less confusing....
Or? Are we talking about a space or a storage? IIRC that is a difference, no?

[the remark this answers is not in the document]
Comment (micbar) [reply internal/http/services/owncloud/ocdav/meta.go:147]: needs to be discussed before we freeze the CS3Apis. StorageID was the property which was already there and we use it as the spaceID

**Pull request merged**